README.md

```diff
@@ -98,23 +99,16 @@ associated with any `opentracing.Span` instance.
 
 #### Deserializing from the wire
 
-```
+```go
     http.HandleFunc("/", func(w http.ResponseWriter, req *http.Request) {
-        // Grab the TraceContext from the HTTP header using the
-        // opentracing helper.
-        reqTraceCtx, err := opentracing.TraceContextFromHeader(
-                req.Header, opentracing.GlobalTracer())
-        var serverSpan opentracing.Span
-        var goCtx context.Context = ...
+		// Join the trace in the HTTP header using the opentracing helper.
+        serverSpan, err := opentracing.JoinTraceFromHeader(
```

Comment:
I'd expect this hook to be most commonly called by integrators, as tracers often bundle an http library and/or plugins that start or resume a trace. Some of these set the operation name while reading from headers, but in many cases they don't because operation name can be changed, specified elsewhere or later.

PS headers in zipkin include trace_id, id, parent id, sampled and (while rarely implemented) the debug flag. Brave has an optional way to send the span name in a header iirc.